lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -2063,6 +2076,10 @@ static bool isExpressionAlwaysInline(Operation *op) {
       isa<SubfieldOp>(op))
     return true;
 
+  // An SV interface modport is a symbolic name that is always inlined.
+  if (isa<sv::GetModportOp>(op))
```

Comment:
This is needed to ensure this op is always inlined, even when it is used multiple types.